sklearn/utils/estimator_checks.py

```diff
@@ -636,6 +637,16 @@ def check_sample_weights_invariance(name, estimator_orig):
                       1, 1, 1, 1, 2, 2, 2, 2], dtype=np.dtype('int'))
         y = enforce_estimator_tags_y(estimator1, y)
 
+        # sample weights greater than 1D raises ValueError
+        sample_weight = [[1, 2]]
+        with pytest.raises(ValueError):
```

Comment:
Sorry in advance about this :)

In the `estimator_checks.py`, we are not using `pytest` (it would force third party using this code to use pytest as well).

Therefore, you need to use `assert_raises` instead of `pytest.raises` (but this is the only place).
Also, it might be useful to use `assert_raises_regex` and match a partial string. It would enforce consistency regarding the error message raised.

Reply:
ok, thank you. I realised there was a soft dependency on `pytest` and the `condo` had a problem installing it.